### ISSUE TITLE
Feature extraction for m3p

### ISSUE THREAD

**Issue by Abhiram4572** [UKPLab/MMT-Retrieval#8]
Hi

The feature extraction instructions for m3p provided generates two .npy files for any given image. 
One contains image region features, and the other contains box coordinates, class logits etc.

However, the script to load image features tried to load .pt files (supports loading .pt files), but not .npy files.
In this case of having .npy files, how shall we proceed?

Kindly help. Looking forward to a positive response.

**Comment by gregor-ge** [comment]
Hi, 

thanks for (trying to) use our project! I took a look at the linked script and it is indeed not quite right. Turns out that I manually changed the script for my purposes (i.e., saving everything in one file and using torch to save it) but I forgot that when making the documentation. Very sorry about that😅

I uploaded my script now to the feature_extraction folder and updated the documentation. Usage is the same still. Let me know if that fixes your problem.
